quicklogic/dummy.eblif

```diff
@@ -0,0 +1,4 @@
+.model top
```

Comment:
This is probably in the wrong location.

Reply:
It wasn't actually needed. I removed it.